cluster/addons/cluster-monitoring/prometheus/prometheus-pvc.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: prometheus
+  namespace: kube-system
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: EnsureExists 
```

Comment:
Do we actually intend users to modify this, i.e. other parts than storage request?